core/connectionlistmodel.cpp

```diff
@@ -7,29 +7,15 @@ ConnectionListModel::ConnectionListModel(QObject *parent) :
 {
 }
 
-QHash <int, QByteArray> ConnectionListModel::roleNames() const
-{
-	QHash<int, QByteArray> roles;
-	roles[AddressRole] = "address";
-	return roles;
-}
-
 QVariant ConnectionListModel::data(const QModelIndex &index, int role) const
 {
 	if (index.row() < 0 || index.row() >= m_addresses.count())
 		return QVariant();
-	if (role != AddressRole)
+	if (role != Qt::DisplayRole)
```

Comment:
Old code with that default parameters was very weird - this seems more logical to me.

If it still works, I'm all for it.

Reply:
I tested it a bit and it seems to work just as well.